.github/workflows/testing.yml

```diff
@@ -46,7 +119,10 @@ jobs:
   comprehensive-tests:
     name: ${{ matrix.name }}
     runs-on: ${{ matrix.os }}
-    needs: initial-tests
+
+    if: ${{ needs.skips-allowed.outputs.SKIP_COMPREHENSIVE == 'false' }}
+    needs: [initial-tests, skips-allowed]
```

Comment:
So if `SKIP_COMPREHENSIVE` is false, this needs both of these... oh, no, right, this runs conditionally only if that var is false, correct?

Reply:
Correct